### PR TITLE
[FIX] l10n_vn: ambiguous account name

### PR DESCRIPTION
**Pull request by davidtranhp**
The name of some accounts is ambiguous and too generic. For example, the account 3388 was named "Others" which is difficult for non Vietnamese accountants to understand. Some foreigners doing business in Vietnam usually get upset with these ambiguous names

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_vn/data/account.account.template.csv [hunk 1]
```diff
@@ -28,7 +28,7 @@
 "chart1533","Instruments for renting",1533,"asset_current","vn_template","False"
 "chart1534","Equipment and spare parts for replacement",1534,"asset_current","vn_template","False"
 "chart1541","Construction contracts",1541,"asset_current","vn_template","False"
-"chart1542","Other products",1542,"asset_current","vn_template","False"
+"chart1542","Other Work-in-progress products",1542,"asset_current","vn_template","False"
```
Comment (lordkrandel) [inline addons/l10n_vn/data/account.account.template.csv:31]:
```suggestion
"chart1542","Other work-in-progress products",1542,"asset_current","vn_template","False"
```

**Review by lordkrandel — CHANGES_REQUESTED** [review]
Hello, generally speaking this PR **LGTM**.
If you can fix the uppercase, I'll then ask for `r+`.

image
image

File: addons/l10n_vn/data/account.account.template.csv (hunk 1 above)
Comment (davidtranhp) [reply addons/l10n_vn/data/account.account.template.csv]: @lordkrandel tks for your review. I dit it according to your comment, and rebased too. Please help review again.

**Comment by lordkrandel** [comment]
LGTM, @qdp-odoo for final review.

**Comment by davidtranhp** [comment]
> LGTM, @qdp-odoo for final review.

@qdp-odoo would you mind reviewing this?

**Comment by qdp-odoo** [comment]
@robodoo r+
@davidtranhp thanks for the contribution

**Pull request closed without merging**